test/cli/iGenVar_cli_test.cpp

```diff
@@ -282,7 +282,7 @@ TEST_F(iGenVar_cli_test, fail_negative_min_var_length)
                                          "--min_var_length -30");
     std::string expected_err
     {
-        "[Error] You gave a negative min_var_length parameter.\n"
+        "[Error] Value parse failed for --min_var_length: Argument -30 could not be parsed as type std::string.\n"
```

Comment:
Why is'nt this `size_t`? Is this a bug of the seqan3 parser?
```suggestion
        "[Error] Value parse failed for --min_var_length: Argument -30 could not be parsed as type size_t.\n"
```

Reply:
Resolved by https://github.com/seqan/seqan3/pull/2836